src/extensions/default/SVGCodeHints/SVGUtils.js

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (c) 2015 Amin Ullah Khan. All rights reserved.
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"), 
+ * to deal in the Software without restriction, including without limitation 
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, 
+ * and/or sell copies of the Software, and to permit persons to whom the 
+ * Software is furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING 
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+ * DEALINGS IN THE SOFTWARE.
+ * 
+ */
+
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, maxerr: 50 */
+/*global define, brackets, $ */
+
+define(function (require, exports, module) {
+    "use strict";
+    
+    // Load dependencies.
+    var TokenUtils  = brackets.getModule("utils/TokenUtils");
+    
+    // Enums of token types.
+    var TokenTag    = 1,
+        TokenAttr   = 2,
+        TokenValue  = 3;
+    
+    // Regex to find whether or not an attribute value is quoted in single or double quotation marks.
+    var regexQuotedValue = /^['"][\W\w]*?['"]?$/;
```

Comment:
`[\W\w]` is everything (like `.`), right?
In that case, you should rather use `.`